packages/node/src/indexer/indexer.manager.ts

```diff
@@ -193,10 +193,14 @@ export class IndexerManager {
         ],
       },
       raw: true,
-    });
+    })) as any;
```

Comment:
don't use any, and if you should remove `raw: true` actually.

Reply:
ok I fixed and used a mapping function instead.